### ISSUE TITLE
add fullscreen mode

### ISSUE THREAD

**Issue by cgahr** [bordaigorl/rmview#152]
Hi,

I'd love to have a fullscreen mode for rmview.

I use rmView to present my slides via my laptop using the remarkable. The nice thing is that I can take notes or explain things directly using the pen.

However, I cannot put rmview into fullscreen, which wastes a bit of space on the bottom and top of the screen.

BR, cgahr

PS: really love rmview!

**Comment by bordaigorl** [comment]
Just pushed a fix on the `devel` branch df796163e4d06790ba0d02e6a13cac48577a364d
It wasn't implemented earlier since in OSX maximizing = fullscreen but an explicit toggle is a good idea anyway.
Which system are you on? If you can confirm it works I'll test it some more and then merge in `vnc`

**Comment by cgahr** [comment]
Just tested it, and fullscreen mode seems to work on Ubuntu. However, you cannot exit it by pressing `esc`, so that's something that is still missing.

**Comment by bordaigorl** [comment]
Yeah, it's a toggle so the same shortcut that makes it full screen also exists it.
On GNOME it should be Ctrl+F11, on KDE just F11 might also work.
The app uses `QKeySequence.FullScreen` as a shortcut the interpretation of which is platform-dependent, see the [Qt docs](https://doc.qt.io/qt-5/qkeysequence.html#standard-shortcuts).
There's always the option of using the context menu.
I can see that ESC could be the instinctive thing to try, I'll see what I can do.
